resources/tools/mdsd/devops/pipeline/stages/impl/generic/modules/Deploy/Deploy.groovy

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
I think, I got the idea of this code but some comments would be nice. It would be good to have a comment explaining how you expect the configurations and modules to be named. In addition, the patching and unpatching of the global configuration is a bit confusing. Sorry, I cannot make more concrete requests for improvements.

Reply:
True, this is some juggling with global namespaces and possible overrides by specific variables. I'd address this in the documentation which I will create for the build dsl.